guides/doc-Quickstart_Guide/master.adoc

```diff
@@ -0,0 +1,74 @@
+include::common/attributes.adoc[]
+
+:imagesdir: common/images
+:numbered:
+ifdef::foreman-el,foreman-deb,katello[]
+:toc:
+:toc-placement: left
+endif::[]
+
+:quickstart-installation-guide:
+:context: {project-context}
+:mode: connected
+:ProductName: {ProjectServer}
+
+= {project-installation-guide-title}
+
+The Foreman installer is a collection of Puppet modules that installs everything required for a full working Foreman setup. It uses native OS packaging (e.g. RPM and .deb packages) and adds necessary configuration for the complete installation.
+
+Components include the Foreman web UI, Smart Proxy, a Puppet server, and optionally Passenger, TFTP, DNS and DHCP servers. It is configurable and the Puppet modules can be read or run in “no-op” mode to see what changes it will make.
+
+include::common/modules/ref_supported-operating-systems.adoc[]
+
+The installation will require 4GB of memory, see {InstallingProjectDocURL}system-requirements_{project-context}[System Requirements] for more information.
+
+The Foreman installer uses Puppet (5 or later required) to install Foreman. This guide assumes that you have a newly installed operating system, on which the installer will setup Foreman, a Puppet server, and the Smart Proxy by default. It’s not advisable to follow the steps below on an existing system, since the installer will affect the configuration of several components.
+
+include::common/modules/proc_configuring-repositories.adoc[leveloffset=+1]
+
+include::common/modules/proc_installing-the-satellite-server-packages.adoc[leveloffset=+1]
+
+== Running the Installer
+
+The installation run is non-interactive, but the configuration can be customized by supplying any of the options listed in foreman-installer --help, or by running foreman-installer -i for interactive mode. More examples are given in the Installation Options section. Adding -v will disable the progress bar and display all changes.
+
+To run the installer, execute:
+
+ifdef::foreman-el,foreman-deb[]
+[options="nowrap" subs="+quotes,attributes"]
+----
+foreman-installer
+----
+endif::[]
+ifdef::katello[]
+[options="nowrap" subs="+quotes,attributes"]
+----
+foreman-installer --scenario katello
+----
+endif::[]
+
+After it completes, the installer will print some details about where to find Foreman and the Smart Proxy. Output should be similar to this:
+
+ifdef::foreman-el,foreman-deb[]
+[options="nowrap" subs="+quotes,attributes"]
+----
+Success!
+* Foreman is running at https://pipe-katello-server-nightly-centos8.wareagle.example.com
+* Foreman Proxy is running at https://pipe-katello-server-nightly-centos8.wareagle.example.com:9090
+
+The full log is at /var/log/foreman-installer/foreman.log
```

Comment:
In RH docs, we don't display output that the user doesn't interact with, but I suppose if this is upstream only, it is fine.

Reply:
If that's considered a best practice I have no problems dropping this. I was trying to mix the existing quickstart guide in the Foreman docs with the new style.